apps/member-profile/vite.config.ts.timestamp-1712340593047-e8a63840544c3.mjs

```diff
@@ -0,0 +1,20 @@
+// vite.config.ts
+import { vitePlugin as remix } from "file:///home/vazqueed/Projects/open-source/oyster/node_modules/@remix-run/dev/dist/index.js";
+import { installGlobals } from "file:///home/vazqueed/Projects/open-source/oyster/node_modules/@remix-run/node/dist/index.js";
+import { defineConfig } from "file:///home/vazqueed/Projects/open-source/oyster/apps/member-profile/node_modules/vite/dist/node/index.js";
+import tsconfigPaths from "file:///home/vazqueed/Projects/open-source/oyster/node_modules/vite-tsconfig-paths/dist/index.mjs";
+installGlobals();
+var vite_config_default = defineConfig((env) => {
+  return {
+    plugins: [remix(), tsconfigPaths()],
+    server: { port: 3e3 },
+    // This is needed to support the top-level await for initializing feature
+    // flags. We only change the target for the server build though, to avoid
+    // any potential issues with browser compatibility.
+    ...env.isSsrBuild && { build: { target: "esnext" } }
+  };
+});
+export {
+  vite_config_default as default
+};
+//# sourceMappingURL=data:application/json;base64,ewogICJ2ZXJzaW9uIjogMywKICAic291cmNlcyI6IFsidml0ZS5jb25maWcudHMiXSwKICAic291cmNlc0NvbnRlbnQiOiBbImNvbnN0IF9fdml0ZV9pbmplY3RlZF9vcmlnaW5hbF9kaXJuYW1lID0gXCIvaG9tZS92YXpxdWVlZC9Qcm9qZWN0cy9vcGVuLXNvdXJjZS9veXN0ZXIvYXBwcy9tZW1iZXItcHJvZmlsZVwiO2NvbnN0IF9fdml0ZV9pbmplY3RlZF9vcmlnaW5hbF9maWxlbmFtZSA9IFwiL2hvbWUvdmF6cXVlZWQvUHJvamVjdHMvb3Blbi1zb3VyY2Uvb3lzdGVyL2FwcHMvbWVtYmVyLXByb2ZpbGUvdml0ZS5jb25maWcudHNcIjtjb25zdCBfX3ZpdGVfaW5qZWN0ZWRfb3JpZ2luYWxfaW1wb3J0X21ldGFfdXJsID0gXCJmaWxlOi8vL2hvbWUvdmF6cXVlZWQvUHJvamVjdHMvb3Blbi1zb3VyY2Uvb3lzdGVyL2FwcHMvbWVtYmVyLXByb2ZpbGUvdml0ZS5jb25maWcudHNcIjtpbXBvcnQgeyB2aXRlUGx1Z2luIGFzIHJlbWl4IH0gZnJvbSAnQHJlbWl4LXJ1bi9kZXYnO1xuaW1wb3J0IHsgaW5zdGFsbEdsb2JhbHMgfSBmcm9tICdAcmVtaXgtcnVuL25vZGUnO1xuaW1wb3J0IHsgZGVmaW5lQ29uZmlnIH0gZnJvbSAndml0ZSc7XG5pbXBvcnQgdHNjb25maWdQYXRocyBmcm9tICd2aXRlLXRzY29uZmlnLXBhdGhzJztcblxuaW5zdGFsbEdsb2JhbHMoKTtcblxuZXhwb3J0IGRlZmF1bHQgZGVmaW5lQ29uZmlnKChlbnYpID0+IHtcbiAgcmV0dXJuIHtcbiAgICBwbHVnaW5zOiBbcmVtaXgoKSwgdHNjb25maWdQYXRocygpXSxcbiAgICBzZXJ2ZXI6IHsgcG9ydDogMzAwMCB9LFxuXG4gICAgLy8gVGhpcyBpcyBuZWVkZWQgdG8gc3VwcG9ydCB0aGUgdG9wLWxldmVsIGF3YWl0IGZvciBpbml0aWFsaXppbmcgZmVhdHVyZVxuICAgIC8vIGZsYWdzLiBXZSBvbmx5IGNoYW5nZSB0aGUgdGFyZ2V0IGZvciB0aGUgc2VydmVyIGJ1aWxkIHRob3VnaCwgdG8gYXZvaWRcbiAgICAvLyBhbnkgcG90ZW50aWFsIGlzc3VlcyB3aXRoIGJyb3dzZXIgY29tcGF0aWJpbGl0eS5cbiAgICAuLi4oZW52LmlzU3NyQnVpbGQgJiYgeyBidWlsZDogeyB0YXJnZXQ6ICdlc25leHQnIH0gfSksXG4gIH07XG59KTtcbiJdLAogICJtYXBwaW5ncyI6ICI7QUFBNFcsU0FBUyxjQUFjLGFBQWE7QUFDaFosU0FBUyxzQkFBc0I7QUFDL0IsU0FBUyxvQkFBb0I7QUFDN0IsT0FBTyxtQkFBbUI7QUFFMUIsZUFBZTtBQUVmLElBQU8sc0JBQVEsYUFBYSxDQUFDLFFBQVE7QUFDbkMsU0FBTztBQUFBLElBQ0wsU0FBUyxDQUFDLE1BQU0sR0FBRyxjQUFjLENBQUM7QUFBQSxJQUNsQyxRQUFRLEVBQUUsTUFBTSxJQUFLO0FBQUE7QUFBQTtBQUFBO0FBQUEsSUFLckIsR0FBSSxJQUFJLGNBQWMsRUFBRSxPQUFPLEVBQUUsUUFBUSxTQUFTLEVBQUU7QUFBQSxFQUN0RDtBQUNGLENBQUM7IiwKICAibmFtZXMiOiBbXQp9Cg==
```

Comment:
I think you forgot to remove this file, it doesn't seem relevant to the icon changes